src/dtos/lookup-table.ts

```diff
@@ -1,4 +1,4 @@
-export class LookupTableDTO {
+export class LookupTable {
```

Comment:
Any reason for breaking with the convention here by removing the DTO suffix? I'm not precious about keeping it but we should either always have it or never have it in that case :)